### PR TITLE
Add autotest

### PR DESCRIPTION
**Pull request by BBuf**
为下面的API添加autotest接口或去掉一些多余的没有代表性的测试样例：
- [x] oneflow.meshgrid
- [x] oneflow.pow
- [x] oneflow.abs 
- [x] oneflow.nn.LogSoftmax
- [x] oneflow.nn.Hardtanh
- [x] oneflow.ceil
- [x] oneflow.nn.BatchNorm2d 
- [x] oneflow.stack 
- [x] oneflow.nn.Softmax 
- [x] 删除PReLU Op注册的多余代码

### CODE REVIEW

File: python/oneflow/test/modules/test_abs.py [hunk 1]
```diff
@@ -17,75 +17,16 @@
...
-
-def _test_abs_tensor_function_backward(test_case, device):
-    np_input = np.random.randn(2, 3).astype(np.float32)
-    input = flow.Tensor(np_input, dtype=flow.float32, requires_grad=True)
-    of_out = input.abs().sum()
-    of_out.backward()
-    np_grad = np.where(np_input > 0, 1, -1)
-    test_case.assertTrue(np.allclose(input.grad.numpy(), np_grad, 1e-05, 1e-05))
-
+from test_util import GenArgList
```
Comment (MARD1NO) [inline python/oneflow/test/modules/test_abs.py:22]: 这个好像没用到

File: python/oneflow/test/modules/test_pow.py [hunk 2]
```diff
@@ -22,92 +22,31 @@
...
+        return torch.pow(x, y)
+
+    @autotest()
+    def test_pow_elementwise_with_random_data(test_case):
+        device = random_device()
+        x = random_pytorch_tensor(ndim=2, dim1=2).to(device)
+        y = random_pytorch_tensor(ndim=2, dim1=2).to(device)
+        return torch.pow(x, y)
+
+    @unittest.skip("pow has a bug")
```
Comment (MARD1NO) [inline python/oneflow/test/modules/test_pow.py:43]: 这个注释不太好，改成not support for broadcast currently

**Review by MARD1NO — APPROVED** [review]

File: python/oneflow/test/modules/test_abs.py (hunk 1 above)
Comment (BBuf) [reply python/oneflow/test/modules/test_abs.py:22]: 删掉之后：  File "/home/zhangxiaoyu/oneflow/python/oneflow/test/modules/test_abs.py", line 23, in <module>
    @flow.unittest.skip_unless_1n1d()
AttributeError: module 'oneflow' has no attribute 'unittest'

File: python/oneflow/test/modules/test_pow.py (hunk 2 above)
Comment (BBuf) [reply python/oneflow/test/modules/test_pow.py:43]: 好的

**Pull request merged**